src/style.css

```
@@ -1 +1,80 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  
+  --color: #00acdc;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  min-width: 1024px;
+}
```

```suggestion
.header {
  display: flex;
  justify-content: space-between;
  align-items: center;
  width: 100%;
  min-width: 1024px;
  padding: 0 50px;
}
```
instead margins on logo and nav